client/src/helpers/methods.js

```diff
@@ -0,0 +1,33 @@
+const titleToURL = function (title) {
+  const titleLower = title.toLowerCase();
+  const titleKebab = titleLower.replaceAll(" ", "-");
+  const url = `https://leetcode.com/problems/${titleKebab}/`;
+  return url;
+};
+// async call to LeetCode api - replace with call to our backend when that works.
+async function getLeetCodeProblem(url) {
```

Comment:
nice!! great choice in improving modularity with centralizing helper methods in a file!